commands/qemu-guest-tools/guesttools.go

```diff
@@ -326,10 +326,6 @@ func (g *guestTools) doExecShell(ID string, command []string, tty bool) {
 		err = pipeCommand(shell, handler)
 	}
 
-	// If starting the shell didn't fail, then we wait for the shell to terminate
-	if err == nil {
-		err = shell.Wait()
-	}
```

Comment:
I moved this into `pipeCommand|pipePty` to ensure that I'm done streaming before I send the exit message and close the websocket...